python/foglamp/services/core/scheduler/scheduler.py

```diff
@@ -860,7 +866,16 @@ def _schedule_first_task(self, schedule, current_time):
             await asyncio.sleep(1)
 
         if self._task_processes:
-            raise TimeoutError("Timeout Error: Could not stop scheduler")
+            task_count = 0
+            for task_id in list(self._task_processes.keys()):
+                try:
+                    task_process = self._task_processes[task_id]
+                    schedule = task_process.schedule
+                    task_count += 1 if schedule.type != Schedule.Type.STARTUP else 0
+                except KeyError:
+                    continue
```

Comment:
what above block is doing; just counting?

Reply:
added the doc string